fuzzers/005-tilegrid/generate_full.py

```diff
@@ -562,18 +529,27 @@ def db_add_bits(database, segments):
                     ("RIOB33_SING", "CLB_IO_CLK"): (42, 2, 4),
                     ("LIOB33_SING", "CLB_IO_CLK"): (42, 2, 4),
                 """
-                entry = {
-                    # (tile_type, block_type): (frames, words, height)
-                    ("CLBLL", "CLB_IO_CLK"): (36, 2, 2),
-                    ("CLBLM", "CLB_IO_CLK"): (36, 2, 2),
-                    ("HCLK", "CLB_IO_CLK"): (26, 1, 1),
-                    ("INT", "CLB_IO_CLK"): (28, 2, 2),
-                    ("BRAM", "CLB_IO_CLK"): (28, 10, None),
-                    ("BRAM", "BLOCK_RAM"): (128, 10, None),
-                    ("DSP", "CLB_IO_CLK"): (28, 2, 10),
-                    ("INT_INTERFACE", "CLB_IO_CLK"): (28, 2, None),
-                    ("BRAM_INT_INTERFACE", "CLB_IO_CLK"): (28, 2, None),
-                }.get((nolr(tile_type), block_type), None)
+
+                is_not_segint = "SEG_INT" not in segment_name
```

Comment:
Why is this special case needed? Why not just use the tile type like the rest of the code?

Reply:
Mainly because there already is an "INT" in the previous entry with a 28 as frames value. I wanted to separate the two different cases

Reply:
Isn't 28 the correct value? INT tiles don't use more than that. Its possible the PS7 configuration is larger, but all of it may not apply to INT

Reply:
Ok, I'll keep 28 then and delete the special case